src/implcontext.c

```diff
@@ -64,3 +64,8 @@ int8_t *getPoWResult(ImplContext *impl_ctx, void *pow_ctx)
 {
     return impl_ctx->getPoWResult(pow_ctx);
 }
+
+uint64_t getHashTimes(ImplContext *impl_ctx, void *pow_ctx)
```

Comment:
Why do you need implementation-specific counter?

Reply:
We need a counter in each `pow_ctx`.

Reply:
I'll set a counter named `hash_count` rather than `hash_times` to avoid misunderstanding in each `pow_ctx`.

And I'm not sure about the meaning of @jserv 's question.

Reply:
I would like to see the methods to minimize changes against existing interface. We might split the functions of given interface into two fields:
* ops (operations)
* info (information)
That is, PoW takes ops after initialization, which expects info.